packages/react-storage/src/components/StorageBrowser/controls/hooks/useDataTable.ts

```diff
@@ -154,7 +154,7 @@ export const useDataTable = (): DataTableProps => {
 
   return {
     headers: mappedHeaders ?? [],
-    isLoading: !!data.isDataRefreshDisabled,
+    isLoading: data.isLoading,
```

Comment:
Nit: destructure `isLoading` and `loadingIndicator`  above and you can just use object shorthand

Reply:
Agreed. Updated to destructure and use shorthand instead